docs/APIReference-AtomicBlockUtils.md

```diff
@@ -24,3 +24,21 @@ insertAtomicBlock: function(
   character: string
 ): EditorState
 ```
+
+### moveAtomicBlockBefore
+
+```
+moveAtomicBlockBefore: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
+
+### moveAtomicBlockAfter
+
+```
+moveAtomicBlockAfter: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
```

Comment:
Ah, I think I understand -- this is essentially following the pattern of `insertAtomicBlock`, assuming that the `SelectionState` specifies the block to move. I guess I find myself not liking that either. :)